perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/ServicesManagerBlImpl.java

```diff
@@ -1032,4 +1063,23 @@ public void setPerunBl(PerunBl perunBl) {
 	public int getDestinationsCount(PerunSession sess) {
 		return getServicesManagerImpl().getDestinationsCount(sess);
 	}
+
+	@Override
+	public void deleteDestination(PerunSession sess, Destination destination) throws DestinationAlreadyRemovedException, RelationExistsException {
+		List<Service> services = getServicesManagerImpl().getServicesFromDestination(destination.getId());
+		if (!services.isEmpty()) {
+			throw new RelationExistsException("Destination has some services assigned.");
```

Comment:
I would probably state, that Destination is used by some services and facilities. Its not like its assigned to the service only and I don't think we refer to it as "assignment" when creating.
Same goes for the javadoc.

Reply:
Fixed